ckanext/ckanext-apicatalog_scheming/ckanext/apicatalog_scheming/templates/scheming/organization/group_form.html

```diff
@@ -21,16 +23,17 @@
     {%- endfor -%}
     {{ form.hidden('csrf-token', h.generate_csrf_token()) }}
 
-    <div class="form-actions">
-        {% block delete_button %}
-        {% if action == 'edit' %}
-          {% if h.check_access('group_delete', {'id': data.id}) and action=='edit'  %}
-            {% set locale = h.dump_json({'content': _('Are you sure you want to delete this Organization?')}) %}
-            <a class="btn btn-danger pull-left" href="{% url_for controller='organization', action='delete', id=data.id %}" data-module="confirm-action" data-module-i18n="{{ locale }}">{% block delete_button_text %}{{ _('Delete') }}{% endblock %}</a>
-          {% endif %}
```

Comment:
This check should exist, sysadmins must have a way to delete the organization. However it should check organization_delete instead.

Reply:
I have now put that back